routes/api/comments/index.js

```diff
@@ -31,66 +35,87 @@ router.get('/:comment_id', (req, res, next) => {
 
 // Get all the comments that have that action_type over them.
 router.get('/action/:action_type', (req, res, next) => {
-  Comment.findByActionType(req.params.action_type).then((comments) => {
-    res.status(200).json(comments);
-  }).catch(error => {
-    next(error);
-  });
+  Comment
+    .findByActionType(req.params.action_type)
+    .then((comments) => {
+      res.status(200).json(comments);
+    })
+    .catch(error => {
+      next(error);
+    });
 });
 
 // Get all the comments that were rejected.
 router.get('/status/rejected', (req, res, next) => {
-  Comment.findByStatus('rejected').then((comments) => {
-    res.status(200).json(comments);
-  }).catch(error => {
-    next(error);
-  });
+  Comment
+    .findByStatus('rejected')
+    .then(comments => {
+      res.status(200).json(comments);
+    })
+    .catch(error => {
+      next(error);
+    });
 });
 
 // Returns back all the comments that are in the moderation queue. The moderation queue is pre or post moderated,
 // depending on the settings. The :moderation overwrites this settings.
 // Pre-moderation:  New comments are shown in the moderator queues immediately.
 // Post-moderation: New comments do not appear in moderation queues unless they are flagged by other users.
 router.get('/status/pending', (req, res, next) => {
-  Setting.getModerationSetting().then(function({moderation}){
-    let moderationValue = req.query.moderation;
-    if (typeof moderationValue === 'undefined' || moderationValue === undefined) {
-      moderationValue = moderation;
-    }
-    Comment.moderationQueue(moderationValue).then((comments) => {
-      res.status(200).json(comments);
+  
+  Setting
+    .getModerationSetting()
+    .then(function({moderation}){
```

Comment:
I'd discourage the use of `function` as much as possible if we aren't using `this` anywhere.